style.css

```diff
@@ -0,0 +1,296 @@
+* {
+    box-sizing: border-box;
+    scroll-behavior: smooth;
+}
+
+body {
+    margin: 0;
+    text-align: center;
+    font-family: Georgia, sans-serif;
+}
+
+nav {
+    width: 100%;
+    height: 80px;
+    position: fixed;
+    top: 0;
+    background-color: #f0f8ff;
+    opacity: 0.4;
+    z-index: 3;
+}
+
+nav img {
+    margin-left: 5%;
+    float: left;
+    height: 80px;
+    border-radius: 23px;
+}
+
+.navRight {
+    float: right;
+    margin-right: 20%;
+    padding-top: 5px;
+}
+
+.dropdown {
+    float: right;
+    margin-right: 20%;
+    margin-top: 40px;
+    display: none;
+    background-image: url("button.png");
+    background-size: 100%;
+    height: 5px;
+    width: 5px;
+    padding: 16px;
+}
+
+.dropdown .navText {
+    display: none;
+}
+
+nav li {
+    display: inline-block;
+    margin-top: 30px;
+}
+
+.navText {
+    font-family: "Arial Rounded MT Bold", sans-serif, "Times New Roman";
+    font-weight: bold;
+    text-transform: uppercase;
+    margin-right: 50px;
+    color: #000000;
+    text-decoration: none;
+    border-bottom: 3px solid transparent;
+    padding: 10px;
+    font-size: 13px;
+    transition: 0.5s ease-in-out;
+}
+
+.dropdown:hover .navText {
+    display: block;
+    margin-top: 10px;
+}
+
+
+.navText:hover {
+    border-color: #000000;
+}
+
+.pageOne {
+    height: 740px;
+    opacity: 0.6;
+    background-image: url("tree.png");
+    background-size: cover;
+    background-position: center;
+
+}
+
+.home {
+    width: 80%;
+    margin: 0 auto;
+    padding-top: 180px;
+    text-align: center;
+}
+
+.home h1 {
+    font-size: 3.5rem;
+    margin-bottom: -5%;
+    letter-spacing: 1px;
+    transition: 0.8s;
+}
+
+.home h2 {
+    font-size: 2.5rem;
+    transition: 0.8s;
+}
+
+.home .button {
+    display: inline-block;
+    background: transparent;
+    width: 30%;
+    padding: 1em;
+    margin-top: 20px;
+    font-size: 18px;
+    font-weight: bold;
+    border: 3px solid #000000;
+    color: #000000;
+    text-transform: uppercase;
+    text-decoration: none;
+    transition: 0.5s ease-in-out;
+    border-radius: 20px;
+}
+
+.home .button:hover {
+    background-color: #ffffff;
+    color: #000000;
+    border: #ffffff;
+}
+
+.me {
+    width: 80%;
+    margin: 0 auto;
+    padding-top: 60px;
+    text-align: center;
+}
+
+.pageTwo {
+    height: 740px;
+    background-image: radial-gradient(circle, #ffffff, #000000);
+}
+
+.me h1 {
+    font-size: 2.5rem;
+    border-bottom: 3px solid #000000;
+    transition: 0.8s;
```

Comment:
transition 0.8s is repeated alot through your code. put it in its own separate class and add to html elements where necessary

Reply:
doesnt work for h1s or other items unless i directly put the transition into their specific css